acrobat/scripts/scripts.js

```diff
@@ -18,6 +18,10 @@ document.querySelectorAll('a').forEach((p, idx) => {
   }
 });
 
+document.querySelectorAll('a[href*="susi-"]').forEach((el) => {
+  el.style.display = 'none';
```

Comment:
We can set this as the default in `styles.css`.  
Instead of `display : none` let's do:
```
  pointer-events: none;
  cursor: default;
  ```
we can remove these styles in`handleImsSusi()`

Reply:
resolved: https://github.com/adobecom/dc/pull/819/commits/98edddaaf1edd26392871a6e961d3625089cf653